### PR TITLE
Use old JAXB artifacts

### PR DESCRIPTION
**Pull request by runeflobakk**
To ensure the JAXB facilities (generated code and marshallers) can be living alongside any jakarta.xml.bind namespaced JAXB in newer Java applications.

This (hopefully) ensures that we are using JAXB API and runtime artifacts which have _other_ Maven coordinates than the newer Jakarta-branded artifacts, which started out with still containing old `javax.xml.bind` package, but in later versions changed this to `jakarta.xml.bind`. If dependees use newer Jakarta JAXB with the proper package, this will break the use of this library if it is depending on the Jakarta-artifacts with the javax.xml.bind package.

It's a mess.

https://stackoverflow.com/a/72151763

### CODE REVIEW

File: jaxb/pom.xml [hunk 1]
```diff
@@ -111,6 +132,12 @@
             <version>2.0.6</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>no.digipost</groupId>
+            <artifactId>jul-to-slf4j-junit-extension</artifactId>
+            <version>1.0</version>
+            <scope>test</scope>
+        </dependency>
```
Comment (runeflobakk) [inline jaxb/pom.xml:140]: https://github.com/digipost/jul-to-slf4j-junit-extension

File: jaxb/pom.xml [hunk 2]
```diff
@@ -26,20 +42,25 @@
                 <type>pom</type>
                 <scope>import</scope>
             </dependency>
+            <dependency>
+                <groupId>no.digipost</groupId>
+                <artifactId>jaxb-resolver-com.sun.xml.bind-bom</artifactId>
+                <version>0.1-RC2</version>
+                <type>pom</type>
+                <scope>import</scope>
+            </dependency>
```
Comment (runeflobakk) [inline jaxb/pom.xml:51]: https://github.com/digipost/jaxb-resolver-com.sun.xml.bind

**Review by simenstoa — APPROVED** [review]
Nice! Tar gjerne en live oppsummering av dette på kontoret også 😄

**Comment by runeflobakk** [comment]
Absolutt! Her er det _mye_ gøy!

**Pull request merged**